inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/AbstractSourceOperator.java

```diff
@@ -160,6 +160,14 @@ public void updateOpt(SourceRequest request, Integer groupStatus, String operato
         entity.setVersion(entity.getVersion() + 1);
         entity.setModifier(operator);
         entity.setModifyTime(new Date());
+
+        // Re-issue task if necessary
+        if (GroupStatus.forCode(groupStatus).equals(GroupStatus.CONFIG_SUCCESSFUL)) {
```

Comment:
What if the GroupStatus is failed?

Reply:
Then we need to reset status to NEW. Fixed.